### ISSUE TITLE
 [feature request] implements highlighting changes on the fly.

### ISSUE THREAD

**Issue by zw963** [syohex/emacs-git-gutter#126]
as describe in [this](https://github.com/dgutov/diff-hl/blob/master/diff-hl-flydiff.el) file belongs to
diff-hl mode

```
;; This mode enables diffing on-the-fly (i.e. without saving the buffer first)
```

Thanks

**Comment by syohex** [comment]
It is already implemented. Please see https://github.com/syohex/emacs-git-gutter#live-updating

**Comment by zw963** [comment]
Hi, I think it is not the really real-time update like diff-hl even I set 
`git-gutter:update-interval` to 0.1.

If you need a animation for this, tell me and I will do it.

**Comment by syohex** [comment]
Ah, I mistook argument of `run-with-idle-timer`. It has been fixed at #128. Please check latest version.

**Comment by zw963** [comment]
It worked,  experience is more better now. 

Thanks.
